test/programs/annotation-items/main.ts

```diff
@@ -0,0 +1,34 @@
+interface MyObject {
+  /**
+   * @items {"type":"integer"}
+   */
+  a: number[];
```

Comment:
Why not use this pattern? 

```ts
{
  a: Integer[];
}

/**
 * @type integer
 */
type Integer = number;
```

Reply:
I haven't gotten that to work, for me it always comes through as just a plain number.

I couldn't find a test case showing that this is possible, and when I added one, the test failed to get the desired result.

Reply:
Has that worked for you?  I wonder if it's something that stops working in TypeScript 3.7

Reply:
It works in https://github.com/vega/ts-json-schema-generator. Anyway, I think it's the right way to support item typings so I'll close this pull request.